pkg/kv/kvserver/rangefeed/metrics.go

```diff
@@ -25,11 +25,18 @@ var (
 		Measurement: "Nanoseconds",
 		Unit:        metric.Unit_NANOSECONDS,
 	}
+	metaRangeFeedOverBudgetEvents = metric.Metadata{
+		Name:        "kv.rangefeed.over_budget_events",
+		Help:        "Number of events that exceed whole allocated budget in size",
```

Comment:
I added a metric for number of failures to get memory budget. As for the full budget it would be visible from BytesMonitor that range feeds will borrow from.